src/components/Icon/Icon.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
+import * as Chakra from '@chakra-ui/core';
+
+export enum IconSize {
```

Comment:
🌹   `type IconSize = 's' | 'm' | 'l'`, is a bit DRYer (:

Reply:
Agree, but if we want to stay consistent with the other components, it should be an `enum`

Reply:
I absolutely respect this argument. But I personally believe that consistency shouldn't stand in the way to better architecture as long as it's properly documented (we're using TS, so it is documented), as it's all still dynamically evolving.
That is just a suggestion ...